client/src/pages/FiltersPage/FiltersPage.js

```diff
@@ -7,116 +7,117 @@ import React from 'react';
-            profile={tina}
-            name="Bianca's Radishes"
-            location="Berkeley, CA"
-            tags={['vegetables', 'fruit']}
-          />
-          <Card
-            profile={tina}
-            name="Bianca's Radishes"
-            location="Berkeley, CA"
-            tags={['vegetables', 'fruit']}
-          />
-          <Card
-            profile={tina}
-            name="Bianca's Radishes"
-            location="Berkeley, CA"
-            tags={['vegetables', 'fruit']}
-          />
-          <Card
-            profile={tina}
-            name="Bianca's Radishes"
-            location="Berkeley, CA"
-            tags={['vegetables', 'fruit']}
-          />
-          <Card
-            profile={tina}
-            name="Bianca's Radishes"
-            location="Berkeley, CA"
-            tags={['vegetables', 'fruit']}
-          />
-          <Card
-            profile={tina}
-            name="Bianca's Radishes"
-            location="Berkeley, CA"
-            tags={['vegetables', 'fruit']}
-          />
-          <Card
-            profile={tina}
-            name="Bianca's Radishes"
-            location="Berkeley, CA"
-            tags={['vegetables', 'fruit']}
-          />
+          <div className="list-mode">
+            <Card
+              profile={tina}
+              name="Bianca's Radishes"
+              location="Berkeley, CA"
+              tags={['vegetables', 'fruit']}
+            />
+            <Card
+              profile={tina}
+              name="Bianca's Radishes"
+              location="Berkeley, CA"
+              tags={['vegetables', 'fruit']}
+            />
+            <Card
+              profile={tina}
+              name="Bianca's Radishes"
+              location="Berkeley, CA"
+              tags={['vegetables', 'fruit']}
+            />
+            <Card
+              profile={tina}
+              name="Bianca's Radishes"
+              location="Berkeley, CA"
+              tags={['vegetables', 'fruit']}
+            />
+            <Card
+              profile={tina}
+              name="Bianca's Radishes"
+              location="Berkeley, CA"
+              tags={['vegetables', 'fruit']}
+            />
+            <Card
+              profile={tina}
+              name="Bianca's Radishes"
+              location="Berkeley, CA"
+              tags={['vegetables', 'fruit']}
+            />
+            <Card
+              profile={tina}
+              name="Bianca's Radishes"
+              location="Berkeley, CA"
+              tags={['vegetables', 'fruit']}
+            />
+          </div>
         </div>
       );
     } else {
       return (
-        <div className="filters-page-map">
+        <div className="map-mode">
           <Map
-            center={[50.879, 4.6997]}
+            center={markers[0]}
             zoom={12}
             twoFingerDrag={true}
             provider={mapTilerProvider}
           >
             {Object.keys(markers).map(key => (
```

Comment:
nrather than using `Object.keys` on an array, just call `map` directly on the array and add a second parameter to the callback i.e.
```javascript
markers.map((anchor, index) => {
  // ...
```
then set `payload` and `key` equal to `index` and `anchor` equal to `anchor`